src/devices/bosch.ts

```diff
@@ -908,14 +917,90 @@ const definitions: Definition[] = [
         vendor: 'Bosch',
         description: 'Smoke alarm detector',
         fromZigbee: [fz.battery, fz.ias_smoke_alarm_1],
-        toZigbee: [],
+        toZigbee: [{
+            key: ['intruder_alarm_state', 'smoke_alarm_state', 'boschSmokeDetectorSiren', 'command'],
```

Comment:
Why is `'boschSmokeDetectorSiren', 'command'` needed here?

Reply:
I believe it is not necessary. I'll remove it and re check.